python/cuml/tsa/arima.pyx

```diff
@@ -527,12 +613,12 @@ def fit(y,
             The ARIMA order (p, d, q)
     mu0 : array-like
           Array of trend-parameter estimates. Only used if `d>0`.
-    ar_params0 : List of arrays
-                 List of AR parameter-arrays, one array per series,
-                 each series has `p` parameters.
-    ma_params0 : List of arrays
-                 List of MA parameter-arrays, one array per series,
-                 each series has `q` parameters.
+    ar_params0 : np.ndarray
+                 AR parameters, shape (p, num_batches), column-major
+                 (as returned by estimate_x0)
+    ma_params0 : np.ndarray
+                 MA parameters, shape (q, num_batches), column-major
+                 (as returned by estimate_x0)
```

Comment:
Similar to above, maybe we can consider using `input_to_host_array` to be able to accept either row or column major. Feel free to make it a github issue for a follow up if you feel that's better to not delay this PR.

Reply:
Done, thanks for pointing that out